archeo/functions.php

```diff
@@ -63,119 +60,5 @@ function archeo_styles() {
 
 add_action( 'wp_enqueue_scripts', 'archeo_styles' );
 
-if ( ! function_exists( 'archeo_editor_styles' ) ) :
-
-	/**
-	 * Enqueue editor styles.
-	 *
-	 * @since Archeo 1.0
-	 *
-	 * @return void
-	 */
-	function archeo_editor_styles() {
-
-		// Add styles inline.
-		wp_add_inline_style( 'wp-block-library', archeo_get_font_face_styles() );
-
-	}
-
-endif;
-
-add_action( 'admin_init', 'archeo_editor_styles' );
-
-
-if ( ! function_exists( 'archeo_get_font_face_styles' ) ) :
-
-	/**
-	 * Get font face styles.
-	 * Called by functions archeo_styles() and archeo_editor_styles() above.
-	 *
-	 * @since Archeo 1.0
-	 *
-	 * @return string
-	 */
-	function archeo_get_font_face_styles() {
-
-		return "
-		@font-face{
-			font-family: 'Chivo';
-			font-weight: 100;
-			font-style: normal;
-			font-stretch: normal;
-			font-display: swap;
-			src: url('" . get_theme_file_uri( 'assets/fonts/Chivo-Thin.woff2' ) . "') format('woff2');
-		}
-		@font-face{
-			font-family: 'Chivo';
-			font-weight: 400;
-			font-style: normal;
-			font-stretch: normal;
-			font-display: swap;
-			src: url('" . get_theme_file_uri( 'assets/fonts/Chivo-Regular.woff2' ) . "') format('woff2');
-		}
-		@font-face{
-			font-family: 'Chivo';
-			font-weight: 700;
-			font-style: normal;
-			font-stretch: normal;
-			font-display: swap;
-			src: url('" . get_theme_file_uri( 'assets/fonts/Chivo-Bold.woff2' ) . "') format('woff2');
-		}
-		@font-face{
-			font-family: 'Chivo';
-			font-weight: 100;
-			font-style: italic;
-			font-stretch: normal;
-			font-display: swap;
-			src: url('" . get_theme_file_uri( 'assets/fonts/Chivo-ThinItalic.woff2' ) . "') format('woff2');
-		}
-		@font-face{
-			font-family: 'Chivo';
-			font-weight: 400;
-			font-style: italic;
-			font-stretch: normal;
-			font-display: swap;
-			src: url('" . get_theme_file_uri( 'assets/fonts/Chivo-Italic.woff2' ) . "') format('woff2');
-		}
-		@font-face{
-			font-family: 'Chivo';
-			font-weight: 700;
-			font-style: italic;
-			font-stretch: normal;
-			font-display: swap;
-			src: url('" . get_theme_file_uri( 'assets/fonts/Chivo-BoldItalic.woff2' ) . "') format('woff2');
-		}
-		";
-
-	}
-
-endif;
-
-if ( ! function_exists( 'archeo_preload_webfonts' ) ) :
-
-	/**
-	 * Preloads the main web font to improve performance.
-	 *
-	 * Only the main web font (font-weight: 100,400, font-style: normal) is preloaded here since that font is always relevant. 
-	 * The other fonts are only needed if the user changed style or weight of the fonts,
-	 * and therefore preloading it would in most cases regress performance when that font would otherwise not be loaded
-	 * at all.
-	 *
-	 * @since Archeo 1.0
-	 *
-	 * @return void
-	 */
-	function archeo_preload_webfonts() {
```

Comment:
We might still need this.

Reply:
Yeah it looks like we still need this to preload the fonts.